apps/modeler/CMakeLists.txt

```diff
@@ -6,19 +6,13 @@ set(REASON "")
 
 # QVTK?
 if(NOT HAVE_QVTK)
-  set(DEFAULT AUTO_OFF)
+  set(DEFAULT OFF)
   set(REASON "VTK was not built with Qt support.")
-elseif(NOT ${DEFAULT} STREQUAL "AUTO_OFF")
-  set(DEFAULT TRUE)
-  set(REASON)
+else()
+  set(DEFAULT ON)
```

Comment:
This would change the behaviour, right? Previously, the default was always FALSE

Reply:
Changed back to default off.